### PR TITLE
Fix support of Python 3.9+

### PR DESCRIPTION
**Pull request by vvaadim**
fixes asyncio errors on latest versions of Python
https://github.com/jupyterhub/jupyterhub/pull/3298/commits/ed6231d3aa71f86ed059c285460bd051825af4b0

### CODE REVIEW

File: torrent_client/algorithms/announcer.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def __init__(self, torrent_info: TorrentInfo, our_peer_id: bytes, server_port: i
 
         self._last_tracker_client = None
         self._more_peers_requested = asyncio.Event()
-        self._task = None  # type: Optional[asyncio.Task]
+        self._task = None  # type: Optional[]
```
Comment (borzunov) [inline torrent_client/algorithms/announcer.py:23]: What's the purpose of this change?

Please revert it if it was not intended.

Comment (borzunov) [inline torrent_client/algorithms/announcer.py]: What's the purpose of this change?

If it's not intended, please revert it.

**Review by borzunov — COMMENTED** [review]
Hi @vvaadim, thanks for the fix!

Could you please explain the change in the type annotation?

File: torrent_client/algorithms/announcer.py (hunk 1 above)
Comment (vvaadim) [reply torrent_client/algorithms/announcer.py]: Sorry for waiting that long,
It was not intended, I will revert it.

Comment (vvaadim) [reply torrent_client/algorithms/announcer.py]: It should be good now.

Comment (borzunov) [reply torrent_client/algorithms/announcer.py]: Thanks!

**Comment by borzunov** [comment]
Merged it, thanks again for spotting that!
